homeassistant/components/plex/server.py

```diff
@@ -34,11 +40,15 @@ class PlexServer:
     def __init__(self, server_config, options=None):
         """Initialize a Plex server instance."""
         self._plex_server = None
+        self._clients = {}
+        self._sessions = {}
         self._url = server_config.get(CONF_URL)
         self._token = server_config.get(CONF_TOKEN)
         self._server_name = server_config.get(CONF_SERVER)
         self._verify_ssl = server_config.get(CONF_VERIFY_SSL, DEFAULT_VERIFY_SSL)
         self.options = options
+        self.add_media_player_callback = None
+        self.sensor = None
```

Comment:
Platform entities are a platform concern and should not leak outside the platform. We normally use our dispatch helper to signal the correct entity from a central place.